ietf/doc/views_doc.py

```diff
@@ -901,17 +902,28 @@ def document_email(request,name):
                  )
 
 
-def document_history(request, name):
-    doc = get_object_or_404(Document, docalias__name=name)
-    top = render_document_top(request, doc, "history", name)
-
+def get_diff_revisions(request, name, doc):
     # pick up revisions from events
     diff_revisions = []
 
-    diffable = [ name.startswith(prefix) for prefix in ["rfc", "draft", "charter", "conflict-review", "status-change", ]]
+    diffable = [
```

Comment:
Trivial, but would be clearer to me if `diffable` were a Boolean - move the `any()` here?